Gruntfile.js

```diff
@@ -333,18 +328,18 @@ module.exports = function(grunt) {
   grunt.registerTask('update', ['bowercopy', 'sed']);
   // "devserver" serves editor code directly from the src directory.
   grunt.registerTask('devserver',
-      ['proxymessage', 'express:dev', 'watch']);
+      ['proxymessage', 'express:dev', 'browserify:server', 'watch']);
   // "devserver" serves editor code directly from the src directory.
   grunt.registerTask('sdevserver',
-      ['proxymessage', 'express:sdev', 'watch']);
+      ['proxymessage', 'express:sdev', 'browserify:server', 'watch']);
   // "devserver" serves editor code directly from the src directory.
   grunt.registerTask('testserver',
-      ['proxymessage', 'express:localtest', 'watch']);
+      ['proxymessage', 'express:localtest', 'browserify:server', 'watch']);
```

Comment:
To use `watchify`, simply just set `watch: true` and `keepalive: true` in the `grunt-browserify` options. `devserver`, `sdevserver` and `testserver` use the same `browserify:server` task. Which will watch the source files and do auto-recompile whenever they are modified. 

There is no way to serve source using Browserify. They must be compiled. 

The difference between these 3 servers is that they use different server configs.